src/config/datamodel/EdFi.DmsConfigurationService.DataModel/Application/ApplicationInsertCommand.cs

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.DmsConfigurationService.DataModel.Application
+{
+    public class ApplicationInsertCommand
```

Comment:
We have ApplicationInsertResponse, should we consider renaming to ApplicationInsertRequest ?

Reply:
I was going for the CQRS naming pattern where broadly "C" stands for Commands to modify the state of things and "Q" is query.